corehq/apps/reports/standard/cases/basic.py

```diff
@@ -172,6 +180,9 @@ def case_owners(self):
                 demo=HQUserType.DEMO_USER in user_types,
                 commtrack=HQUserType.COMMTRACK in user_types,
             )
+            if HQUserType.DEACTIVATED in user_types:
+                deactivated_owner_ids = self.get_deactivated_owner_ids()
+                special_owner_ids += deactivated_owner_ids
```

Comment:
shouldn't this only include 'deactivated special owners' and not ALL deactivated users?

Reply:
I just clarified the logic here to maybe better illustrate what is happening (I'm no longer lumping `deactivated_owner_ids` in with `special_owner_ids`, which did what I wanted but was confusing to read). Does this answer the question?

Reply:
I think all deactivated users, actually - "special" owners basically means anything that's not a mobile user, or now, an active mobile user.  Although I do think this logic should then live in `get_special_owner_ids`.

Reply:
Actually, I ended up deleting this (https://github.com/dimagi/commcare-hq/pull/21564/commits/5e033a668409f42280e067e43bfef5815d27dec8) because [Deactivated Mobile Workers] is no longer a filter in the case list (which, to be clear, it isn't on hq right now, it just was in an earlier iteration of this pr).